.prettierignore

```diff
@@ -0,0 +1,35 @@
+node_modules
+.pnp
+.pnp.js
+coverage
+
+.changeset
+
+*.md
+*.mdx
+*.yml
+*.yaml
```

Comment:
Why?

Reply:
me and @markflorkowski found prettier was aggressively formatting all these. I can add these back in if we want

Reply:
now that I thought of it a bit more, maybe just ignore `.github` and `.changeset` and don't restrict by filetypes